liblangutil/SourceReferenceFormatterHuman.cpp

```diff
@@ -96,9 +96,9 @@ void SourceReferenceFormatterHuman::printSourceLocation(SourceReference const& _
 
 		// line 2:
 		frameColored() << line << " |";
-		m_stream << ' ' << _ref.text.substr(0, _ref.startColumn);
-		highlightColored() << _ref.text.substr(_ref.startColumn, locationLength);
-		m_stream << _ref.text.substr(_ref.endColumn) << '\n';
+		m_stream << ' ' << _ref.text.substr(0, static_cast<size_t>(_ref.startColumn));
+		highlightColored() << _ref.text.substr(static_cast<size_t>(_ref.startColumn), locationLength);
```

Comment:
I think the failing of the tests must somehow be related to these changes

Reply:
This looks fine to me.